app/models/story_parser.rb

```diff
@@ -319,11 +368,11 @@ def set_work_attributes(work, location = "", options = {})
       if chapter.content.length > ArchiveConfig.CONTENT_MAX
         # TODO: eventually: insert a new chapter
         chapter.content.truncate(ArchiveConfig.CONTENT_MAX, omission: "<strong>WARNING: import truncated automatically because chapter was too long! Please add a new chapter for remaining content.</strong>", separator: "</p>")
+      elsif chapter.content.length == 0
+        raise Error, "Chapter #{chapter.position} of '#{work.title}' is empty."
```

Comment:
Nitpick: Elsewhere we use double quotes around the title. Also, would "blank" make more sense to a layperson?